### DISCUSSION TITLE
OpenDTU bekommt keine Verbindung zum Wechselrichter HMT /no connection to inverter

### DISCUSSION THREAD

**Discussion by 7-er** [tbnobody/OpenDTU#1774] [Q&A]
- never got a connection between OpenDTU and inverter. HowTo debug?
- 
Seit gestern habe ich eine fertig gekaufte Box mit OpenDTUonBattery. 
Der WR sieht sie und blinkt schnell grün. 
Mit Frequenz und Feldstärke habe erfolglos probiert. Doch in der Konsole kommt niemals eine Antwort vom WR. 
Mir ist aufgefallen, dass  OpenDTU schwerfällig auf Frequenzänderungen reagiert, so konnte ich nicht zuverlässig experimentieren. Leider steht mir weder eine originale DTU noch ein weiterer WR zur Verfügung. Wie komme ich mit der Fehlersuche weiter? 

Liebe Grüße aus dem Alten Land
Siemer

**Comment by MetaChuh** [comment]
>Seit gestern habe ich eine fertig gekaufte Box mit OpenDTUonBattery.
>never got a connection between OpenDTU and inverter. HowTo debug?

@7-er 

bitte fragen zu OpenDTU-OnBattery direkt im OpenDTU-OnBattery projekt stellen.
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions

>gekaufte Box mit OpenDTUonBattery

wende dich ebenfalls an den verkäufer, falls du befürchtest, dass es ein hardware problem sein kann

**Reply by 7-er** [reply]
@MetaChuh : Die Verbindungsprobleme haben mit OnBattery nichts zu tun. Sie sind hier als Dauerproblem zu sehen. 
Es wäre schön, wenn jemand mal den Handshake zwischen OpenDTU und WR erklärt. 
Der Verkäufer ist informiert und rät zu Geduld.
